### DISCUSSION TITLE
Does Hearts and Minds takes advantage of additionnal MODs?

### DISCUSSION THREAD

**Discussion by STyx2909** [Vdauphin/HeartsAndMinds#1492] [General]
This exact one figure:
It is a Hunter Rifles MOD pack. Seems to me it would be the MOD of choice to bring population weapons at first.
https://steamcommunity.com/sharedfiles/filedetails/?id=1878487430

So, if I add this MOD, will population pick randomly among them or not?
If so, how do I do?

**Comment by OverlordZorn** [comment]
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L396

HAM can be customized very easily.
to just add your own stuff and take the time and read through this file in general, a lot of stuff you can adapt here.
